test/ui-test/testSuites/suite_onboarding/tst_signUpNegativeCases/test.feature

```diff
@@ -0,0 +1,93 @@
+      | tester124 | bad2!s        | TesTEr16843/!@01 |
+
+  Scenario Outline: The user cannot sign up with incorrect confirmation-again password
+    Given the user inputs username "<username>"
+    And the user inputs the following "<password>" with ui-component "onboarding_newPsw_Input"
+    And the user inputs the following "<password>" with ui-component "onboarding_confirmPsw_Input"
+    And the user clicks on the following ui-component "onboarding_create_password_button"
+    When the user inputs the following "<wrongpassword>" with ui-component "onboarding_confirmPswAgain_Input"
+    Then the following ui-component "onboarding_finalise_password_button" is not enabled
+
+    Examples:
+      | username  | wrongpassword   | password         |
+      | tester123 | Invalid34       | TesTEr16843/!@00 |
+      | tester123 | TesTEr16843/!@) | TesTEr16843/!@01 |
+
+  Scenario Outline: The user cannot finish Sign Up and Sign In process with wrong password format in both new password and confirmation input
+    Given the user inputs username "<username>"
+    When the user inputs the following "<wrongpassword>" with ui-component "onboarding_newPsw_Input"
+    And the user inputs the following "<wrongpassword>" with ui-component "onboarding_confirmPsw_Input"
+    Then the following ui-component "onboarding_create_password_button" is not enabled
+
+    Examples:
+      | username  | wrongpassword   |
+      | tester123 | Invalid34       |
+
+  Scenario Outline: The user cannot finish Sign Up and Sign In process with right password format in new password input but incorrect in confirmation password input
+    Given the user inputs username "<username>"
+    And the user inputs the following "<password>" with ui-component "onboarding_newPsw_Input"
+    When the user inputs the following "<wrongpassword>" with ui-component "onboarding_confirmPsw_Input"
+    Then the following ui-component "onboarding_create_password_button" is not enabled
+
+    Examples:
+      | username  | wrongpassword   | password         |
+      | tester123 | Invalid34       | TesTEr16843/!@00 |
+      | tester123 | TesTEr16843/!@) | TesTEr16843/!@01 |
+
+  Scenario Outline: The user cannot finish Sign Up and Sign In process with incorrect confirmation-again password
+    Given the user inputs username "<username>"
+    And the user inputs the following "<password>" with ui-component "onboarding_newPsw_Input"
+    And the user inputs the following "<password>" with ui-component "onboarding_confirmPsw_Input"
+    And the user clicks on the following ui-component "onboarding_create_password_button"
+    When the user inputs the following "<wrongpassword>" with ui-component "onboarding_confirmPswAgain_Input"
+    Then the following ui-component "onboarding_finalise_password_button" is not enabled
+
+    Examples:
+      | username  | wrongpassword   | password         |
+      | tester123 | Invalid34       | TesTEr16843/!@00 |
+      | tester123 | TesTEr16843/!@) | TesTEr16843/!@01 |
```

Comment:
do we need those examples? (question valid for the whole suite), is there any triage to do here?

Reply:
No, just only more examples. I will keep only one!